src/extension/injected-script/addEventListener.ts

```diff
@@ -53,7 +53,8 @@ import type { CustomEvents } from './CustomEvents'
                 document.activeElement!.dispatchEvent(e)
                 return undefined!
             } else if (textOrImage.type === 'image') {
-                const xray_binary = globalThis.window.Uint8Array.from(textOrImage.value)
+                const Uint8Array = globalThis.Uint8Array ? globalThis.Uint8Array : globalThis.window.Uint8Array
```

Comment:
Can you give me a reason for this change? IIRC if it is using `globalThis.Uint8Array`, it will be a privileged object (see Firefox sandbox system on MDN) instead of a low-security level object.

Reply:
@zhouhanseng